### PR TITLE
Generate report

### PR DESCRIPTION
**Pull request by awirb**
* function for plotting two datasets of identical shape
* plot difference and cross and longprofile

### CODE REVIEW

File: avaframe/out3SimpPlot/outQuickPlotSimple.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        figure 1: plot raster data for dataset1, dataset2 and their difference
+        figure 2: plot cross and longprofiles for both datasets (ny_loc and nx_loc define location of profiles)
+        -plots are saved to Outputs/out3SimpPlot
+    """
+
+    outDir = os.path.join(avaDir, 'Outputs', 'out3SimpPlot')
+    fU.makeADir(outDir)
+
+    # Load input datasets from input directory
+    datafiles = glob.glob(inputDir+os.sep + '*.asc')
```
Comment (matthiasto) [inline avaframe/out3SimpPlot/outQuickPlotSimple.py:50]: Should it only work with .asc files? Maybe just take all files in the directory?

File: avaframe/out3SimpPlot/outQuickPlotSimple.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    ax3 = fig.add_subplot(133)
+    cmap = cmapdiv
+    im3 = plt.imshow(data1-data2, cmap=cmap,
+                     extent=[0, Lx, 0, Ly], origin='lower', aspect=nx/ny)
+    fig.colorbar(im3, ax=ax3)
+    ax3.set_aspect('auto')
+    ax3.set_xlabel('x [m]')
+    ax3.set_title('Difference ref-sim')
+    fig.tight_layout()
+    fig.savefig(os.path.join(outDir, 'Diff'))
```
Comment (matthiasto) [inline avaframe/out3SimpPlot/outQuickPlotSimple.py:99]: Add subtitle for the 2 left plots with the name of the data ploted

**Review by matthiasto — CHANGES_REQUESTED** [review]
Add some documentation to explain where to put which files

**Review by fso42 — APPROVED** [review]

File: avaframe/out3SimpPlot/outQuickPlotSimple.py (hunk 1 above)
Comment (awirb) [reply avaframe/out3SimpPlot/outQuickPlotSimple.py:47]: I added now .txt files

File: avaframe/out3SimpPlot/outQuickPlotSimple.py (hunk 2 above)
Comment (awirb) [reply avaframe/out3SimpPlot/outQuickPlotSimple.py]: I added the file names as title for the two left figs

**Review by matthiasto — APPROVED** [review]

**Pull request merged**